Formula/idris@2.rb

```diff
@@ -0,0 +1,36 @@
+class IdrisAT2 < Formula
+  desc "Pure functional programming language with dependent types"
+  homepage "https://www.idris-lang.org/"
+  url "https://github.com/idris-lang/Idris2/archive/v0.2.0.tar.gz"
+  sha256 "cc7c48d3fbaa55e536890c9eb4701755081d2b5df9a65f0ecc3e00ee5ccf0a0f"
+
+  keg_only :versioned_formula
+
+  depends_on "coreutils" => :build
+  depends_on "chezscheme"
+
+  def install
+    ENV.deparallelize
+    scheme = Formula["chezscheme"].bin/"chez"
+    system "make", "bootstrap", "SCHEME=#{scheme}", "PREFIX=#{libexec}"
+    system "make", "install", "PREFIX=#{libexec}"
+    idris2_app = libexec/"bin/idris2_app"
+    (bin/"idris2").write_env_script idris2_app/"idris2.so",
```

Comment:
shouldn't this point to the dylib?

Reply:
It is counterintuitive but this is actually a binary. Idris2 outputs binaries with `.so` extension and that is not configurable.

Reply:
Could you make that a comment in the formula? Because someone in the future will think the same thing I just did